test/cleanup.runtime.test.ts

```diff
@@ -0,0 +1,96 @@
+// TODO add env variable to clean up
+
+// import { Role } from "aws-cdk-lib/aws-iam";
+import { FunctionUrlAuthType } from "aws-cdk-lib/aws-lambda";
+import { StringParameter } from "aws-cdk-lib/aws-ssm";
+// import { STS } from "aws-sdk";
+import { $AWS, $SFN, Function, StepFunction } from "../src";
+import {
+  // clientConfig,
+  runtimeTestExecutionContext,
+  runtimeTestSuite,
+  STACK_TAG_KEY,
+} from "./runtime";
+
+// const sts = new STS(clientConfig);
+
+runtimeTestSuite("cleanUp", (test, stack) => {
+  test(
+    "deleter",
+    async (scope) => {
+      // step function which deletes all stacks
+      // lambda url to trigger
+      // set lambda url in ssm parameter
+      // grant access to ?? to call ssm and lambda url
+      // const stackPartition = new Function(
+      //   scope,
+      //   "stackPartition",
+      //   async (stackArns: string[]) => ({
+      //     cleanUpStack: stackArns.find((arn) => arn.includes("/cleanUp/")),
+      //     otherStacks: stackArns.filter((arn) => !arn.includes("/cleanUp/")),
+      //   })
+      // );
```

Comment:
is this no longer needed? Found a functionless way to do it?

Reply:
I was going to wait to delete the cleanup stack, but... found that step functions waits for running jobs to finish anyways, so it s safe to just delete them all.

Forgot to delete the extra code.